packages/cst/src/expressions.candy

```diff
@@ -1,23 +1,18 @@
-use lexer
 use ..declarations
 use ..node
+use ..tokens
 use ..types
 
 public trait /* enum */ CstExpression
 
 
-public data class CstBoolExpression {
-  public let value: CstNode<LiteralToken<Bool>>
-}
-impl CstBoolExpression: CstExpression
-
-public data class CstIntExpression {
-  public let value: CstNode<LiteralToken<Int>>
+public class CstIntExpression {
+  public let value: CstNode<IntLiteralToken>
 }
 impl CstIntExpression: CstExpression
 
 
-public data class CstStringExpression {
```

Comment:
`CstNode<T: Equals & Hash>` can't currently become a data class because its `trailingTrivia: List<WhitespaceToken | CommentToken>` currently doesn't implement `Hash & Equals`